doc.md

```diff
@@ -1790,6 +1790,7 @@ You may instead divide it into multiple lines in between double quotes by escapi
 	ex=01;32:\
 	"
 
+The `ln` entry supports the special value `target`, which will use the link target to select a style. (this mirrors GNU's ls and dircolors behavior)
```

Comment:
I think it might be better to explicitly mention that filename rules apply to the link name and not the target name, otherwise it might be confusing.